<launch>
   <node pkg="bridge_px4" type="GCS" name ="GCS" output="screen">
         <param name="traj_id" value="$(find bridge_px4)/trajectories/square_grp_drone1.csv"/>
         <param name="t_final" value="30.0"/>
   </node>
</launch>
